gen3utils/etl/etl_validator.py

```diff
@@ -312,13 +317,25 @@ def check_mapping_constraints(mappings, model, recorded_errors):
     return recorded_errors
 
 
-def validate_mapping(dictionary_url, mapping_file):
+def validate_mapping(dictionary_url, mapping_file, manifest):
     dictionary, model = init_dictionary(dictionary_url)
     with open(mapping_file) as f:
         mappings = yaml.safe_load(f)
 
+    # If using tube >= 0.4.0 or >= 2020.10, {doc_type}_id fields have a prefixed underscore.
+    # https://github.com/uc-cdis/tube/releases/tag/0.4.0
+    tube_version = get_manifest_version(
+        manifest["versions"], "tube", release_tag_are_branches=False
+    )
+    if tube_version >= version.parse("0.4.0") or tube_version >= version.parse(
```

Comment:
i think this might crash if tube is not in the manifest

Reply:
You're 100% right! The test suite just caught that too. Fixin it now